### PR TITLE
Documents Page

### PR DESCRIPTION
**Pull request by Nakul727**
Close https://github.com/CSSS/csss-site-frontend/issues/49

Hey, this is the documents page, really basic just wanted to commit the work that I did. Also, I was working on the affiliates page and wrote some boilerplate code which I committed because why not. Tell me if you want to remove something or just change it completely because I didn't spend too much time on it. Just didn't want to hold the task for too long so that other people can work on it, which might be better. 

Screenshot 2024-08-17 at 11 14 39 PM

### CODE REVIEW

**Comment by Nakul727** [comment]
I also changed the way footer was exported from the components folder which might cause some merge problems. lmk if you want me to change it back.

**Comment by Nakul727** [comment]
Also I think the issue was changed and documents page was assigned to someone else. I already made the page so it will be good if there is no duplicate work. If they already did, then choose the better design :)

File: src/main/pages/Affiliates.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Page, Footer } from '../components';
+import { useSpring, animated } from 'react-spring';
+
+// Affiliates data
+const affiliates = [
+  {
+    text: 'WiCS',
+    image: '/static/files/affiliates/wics.jpg',
```
Comment (EarthenSky) [inline src/main/pages/Affiliates.js:9]: I notice these images weren't checked in, I think it would be a good idea to serve these from our repo.

**Review by EarthenSky — CHANGES_REQUESTED** [review]
In terms of style this is lovely & consistent! I notably love the alignment of the documents page.

I think we'll want to add more documents to this page eventually (SFSS documents that are important, constitutions from github, etc...), at which time it would be helpful to store them all in a table (maybe even with sortable columns?!). Same with the affliates page, it might be nice to add a bit of text for each affiliate.

But for now, this is perfect!

**Comment by EarthenSky** [comment]
> Also I think the issue was changed and documents page was assigned to someone else. I already made the page so it will be good if there is no duplicate work. If they already did, then choose the better design :)

I'm not sure if you checked in with @micahdbak or myself, but please do next time (& grab the issue by assigning yourself! I think you should have the permissions to do so). If you want a page that someone's already assigned, feel free to ask us too! A lot of the issues haven't been worked on yet, just assigned so people have an idea of what to do.

Ultimately I think this is great, and it luckily seems like @gunmack hadn't started yet, so we're all good with picking your design.

**Comment by gunmack** [comment]
Looks good, just to confirm I did not start on this

File: public/static/files/docs/constitution_july2024.pdf
[the remark this answers is not in the document]
Comment (EarthenSky) [reply public/static/files/docs/constitution_july2024.pdf:1]: We can use https://raw.githubusercontent.com/CSSS/public-docs/dev-update-constitution-policies/constitutions/Constitution_July_2024.pdf and https://raw.githubusercontent.com/CSSS/public-docs/dev-update-constitution-policies/policies/Policies_July_2024.pdf for the time being.

[the remark this answers is not in the document]
Comment (Nakul727) [reply public/static/files/docs/constitution_july2024.pdf:1]: Sounds good! Will update that.

File: src/main/pages/Affiliates.js (hunk 1 above)
Comment (Nakul727) [reply src/main/pages/Affiliates.js:9]: Yeah, these were just temporary images links that I added for the affiliate page. I wanted to remove the code but decided it's worth keeping just to get some suggestions. I have a cool idea on making the page in a sort of expandable card grid ... will try to implement it soon.

**Comment by Nakul727** [comment]
I think it is good to separate the documents and affiliate page issue because I kind of initially told Micah I will do both at the same time. @gunmack if you want to work on the documents page, you can either continue/edit the code or rewrite it with the table format. I can focus on the affiliate page. Lmk what you want to do.

**Comment by Nakul727** [comment]
Accidentally closed it 😅

File: src/main/pages/Affiliates.js (hunk 1 above)
Comment (EarthenSky) [reply src/main/pages/Affiliates.js:9]: sound good, lovely!

**Comment by EarthenSky** [comment]
> I think it is good to separate the documents and affiliate page issue

I think what you have is a pretty good first step for both pages! Let’s merge this PR & you can make a new PR for the changes you (or @gunmack) might want to make.

I can fix the two small issues I mentioned later tonight if you haven’t already.

File: src/main/components/Footer.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import {
 } from '@fortawesome/free-brands-svg-icons';
 import { faEnvelope } from '@fortawesome/free-solid-svg-icons';
 
-const Footer = () => {
+export const Footer = () => {
```
Comment (micahdbak) [inline src/main/components/Footer.js:11]: Gracias <3

**Review by micahdbak — COMMENTED** [review]
This is amazing, I'll update the response from Gabe and get this merged

**Comment by micahdbak** [comment]
(Replaced by https://github.com/CSSS/csss-site-frontend/pull/60)

**Pull request closed without merging**